src/client/entity-editor/name-section/name-section.js

```diff
@@ -224,7 +223,6 @@ class NameSection extends React.Component {
 						</Col>
 					</Row>
 					{
```

Comment:
You can remove the opening and closing { and } which aren't necessary anymore.

Reply:
yes i should !